lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -0,0 +1,11 @@
+<div class="certificate-container">
+    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <div class="certificate-image"></div>
```

Comment:
Why is this being added as a background image to an empty div instead of as an ```<img>```? I think it should be added to the DOM via an image element.